service/s3/s3manager/upload.go

```diff
@@ -480,19 +501,33 @@ func readFillBuf(r io.Reader, b []byte) (offset int, err error) {
 // a regular PutObject request. Multipart requests require at least two
 // parts, or at least 5MB of data.
 func (u *uploader) singlePart(buf io.ReadSeeker) (*UploadOutput, error) {
+	var (
+		r       io.ReadSeeker
+		cleanup func()
+	)
+
+	r = buf
```

Comment:
since `r` is set once, could use `:=` for setting it instead of creating it in the var block above. only `cleanup` would need to be a `var`.